src/commands/appSettings/uploadAppSettings.ts

```diff
@@ -53,6 +53,11 @@ export async function uploadAppSettingsInternal(context: IActionContext, client:
         }
 
         const excludedAppSettings: string[] = [];
+        if (localSettings.Values["AzureWebJobsStorage"] === 'UseDevelopmentStorage=true') {
```

Comment:
Definitely good to have a clarifying comment here about this one. In this case, it's probably good to also cite the issue since it describes the scenario pretty well.

Reply:
Small nit, but I think we already have a constant that could be used here representing `UseDevelopmentStorage=true`

Reply:
He is correct, `localEmulatorConnectionString`.